coderd/coderd.go

```diff
@@ -395,6 +403,10 @@ func New(options *Options) *API {
 			r.Use(apiKeyMiddleware)
 			r.Get("/", entitlements)
 		})
+		r.Route("/licenses", func(r chi.Router) {
+			r.Use(apiKeyMiddleware)
+			r.Mount("/", options.LicenseHandler)
```

Comment:
It seems awkward to mount an entire router when this is a single HTTP handler.

I'm not sure being more specific wouldn't get tiring after a while, but I think it'd reduce confusion in the short term. eg. we call this `postLicense` instead. I'm not a massive fan of that either though... 🤔

Reply:
as discussed on discord, PRs for GET and DELETE are coming in short order.  I'd prefer to keep the number of options down and install a router for all the endpoints.